.github/workflows/visual-regression.yml

```diff
@@ -1,4 +1,4 @@
-name: Visual regression checks
+name: Start Percy build
```

Comment:
I changed the wording here because this task doesn't actually perform any checks, it just instructs Percy to start a build. The Percy GitHub integration is the thing that actually feeds back any build breaking changes

Reply:
OK, but my point about 'who is Percy' still stands! Could we meet halfway with `Start visual regression build`?